proxy/proxy/dial.go

```diff
@@ -25,7 +25,15 @@ import (
 	"golang.org/x/oauth2/google"
 )
 
-const port = 3307
+// The port that CloudSQL expects the client to connect to.
+const Port = 3307
+
+// DelayKeyGenerate (if true) defers the generation of the CertSource's RSA
+// key to the first connection attempt to a Cloud SQL database. This is useful
+// in scenarios when the proxy needs to be created no matter what (e.g. when
+// mounting the FUSE), but may never be used. If this behavior is desired, this
+// var should be set to true before calling proxy.Init.
+var DelayKeyGenerate bool
```

Comment:
This value isn't used anymore, I think we can remove it.

Reply:
Whoops, I thought I'd already done that. Done now.